editor/editor_help.h

```diff
@@ -83,6 +85,36 @@ class FindBar : public HBoxContainer {
 	FindBar();
 };
 
+class EditorHelpImageDownloader : public Control {
+	GDCLASS(EditorHelpImageDownloader, Control);
+
+	RichTextLabel *rtl = nullptr;
+
+	Ref<Texture2D> progress[8]{};
+	Ref<Texture2D> progress_fill[8]{};
+	Ref<Texture2D> progress_error;
```

Comment:
It would be good to use the `ThemeCache` pattern here. See `EditorHelp` and `EditorHelp::_update_theme_item_cache()` for an example.